src/argument_types.py

```diff
@@ -0,0 +1,206 @@
+
+
+class Arguments(BaseModel):
+    """
+    Define arguments
+    """
+
+    lat: float = 0.0
+    long: float = 0.0
+    city: str = "pleasure_point"
+    show_wave: bool = True
+    show_large_wave: bool = False
+    show_uv: bool = True
+    show_height: bool = True
+    show_direction: bool = True
+    show_period: bool = True
+    show_city: bool = True
+    show_date: bool = True
+    show_air_temp: bool = False
+    show_wind_speed: bool = False
+    show_wind_direction: bool = False
+    json_output: bool = False
+    show_rain_sum: bool = False
+    show_precipitation_prob: bool = False
+    unit: Literal["imperial", "metric"] = "imperial"
+    decimal: int = Field(default=1, ge=0)
+    forecast_days: int = Field(default=0, ge=0, le=7)
+    color: str = "blue"
+    gpt: bool = False
+
+
+class ArgumentMappings(BaseModel):
+    """
+    Class for argument mappings with multiple aliases.
+    """
+
+    city: str = Field(
+        default="pleasure_point",
+        description="Name of the city you want weather data from",
+    )
+    show_wave: bool = Field(
+        default=True,
+        description="Show wave information",
+    )
+    show_large_wave: bool = Field(
+        default=False,
+        description="Show large wave information",
+    )
+    show_uv: bool = Field(
+        default=True,
+        description="Show UV information",
+    )
+    show_height: bool = Field(
+        default=True,
+        description="Show wave height information",
+    )
+    show_direction: bool = Field(
+        default=True,
+        description="Show wave direction information",
+    )
+    show_period: bool = Field(
+        default=True,
+        description="Show wave period information",
+    )
+    show_city: bool = Field(
+        default=True,
+        description="Show city information",
+    )
+    show_date: bool = Field(
+        default=True,
+        description="Show date information",
+    )
+    unit: Literal["imperial", "metric"] = Field(
+        default="imperial",
+        description="Set unit system",
+    )
+    json_output: bool = Field(
+        default=False,
+        description="Enable JSON output",
+    )
+    gpt: bool = Field(
+        default=False,
+        description="Enable GPT functionality",
+    )
+    show_air_temp: bool = Field(
+        default=False,
+        description="Show air temperature",
+    )
+    show_wind_speed: bool = Field(
+        default=False,
+        description="Show wind speed",
+    )
+    show_wind_direction: bool = Field(
+        default=False,
+        description="Show wind direction",
+    )
+    show_rain_sum: bool = Field(
+        default=False,
+        description="Show rain sum",
+    )
+    show_precipitation_prob: bool = Field(
+        default=False,
+        description="Show precipitation probability",
+    )
+
+    alias_map: ClassVar[dict[str, list[str]]] = {
+        "city": ["loc", "location"],
+        "show_wave": ["sw", "hide_wave", "hw"],
```

Comment:
Why is show wave mapped to both "sw" (which i assume means "show wave") and "hide_wave"?